content/filterx/function-reference.md

```diff
@@ -358,34 +358,22 @@ See also {{% xref "/filterx/_index.md#delete-values" %}}.
 
 ## unset_empties {#unset-empties}
 
-Deletes ([unsets](#unset)) the empty fields of an object, for example, a JSON object or list. Use the `recursive=true` parameter to delete empty values of inner dicts' and lists' values.
+Deletes ([unsets](#unset)) the empty fields of an object, for example, a JSON object or list. By default, the object is processed recursively, so the empty values are deleted from inner dicts and lists as well. If you set the `replacement` option, you can also use this function to replace fields of the object to custom values.
 
-Usage: `unset_empties(object, recursive=true)`
+Usage: `unset_empties(object, options)`
 
-<!-- FIXME add a before/after example, for recursive and non-recursive cases 
+The `unset_empties()` function has the following options:
 
-            dict = json({"foo": "", "bar": "-", "baz": "N/A", "almafa": null, "kortefa": {"a":{"s":{"d":{}}}}, "szilvafa": [[[]]]});
-            defaults_dict = dict;
-            explicit_dict = dict;
-            unset_empties(defaults_dict);
-            unset_empties(explicit_dict, recursive=true);
+- `ignorecase`: Set to `false` to perform case-insensitive matching. Default value: `true`. Available in Available in {{< product >}} 4.9 and later.
```

Comment:
- `ignorecase`: Set to `false` to perform case-sensitive matching. Default value: `true`. Available in Available in {{< product >}} 4.9 and later.